package.json

```diff
@@ -108,30 +109,21 @@
     "yup": "0.31.1"
   },
   "devDependencies": {
-    "@edx/browserslist-config": "1.2.0",
     "@edx/react-unit-test-utils": "3.0.0",
-    "@edx/reactifex": "^1.0.3",
     "@edx/stylelint-config-edx": "2.3.3",
     "@edx/typescript-config": "^1.0.1",
-    "@openedx/frontend-build": "^14.0.14",
     "@testing-library/jest-dom": "5.17.0",
     "@testing-library/react": "12.1.5",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^13.2.1",
     "@types/lodash": "^4.17.7",
-    "axios": "^0.28.0",
```

Comment:
We shouldn't specify an `axios` version as it's never directly imported. `frontend-platform` wraps it for us and determines which version to use.